.github/actions/e2e-builder/exec-tests.sh

```diff
@@ -114,6 +117,10 @@ export KAMEL_INSTALL_OPERATOR_IMAGE=${CUSTOM_IMAGE}:${CUSTOM_VERSION}
 # (see kamel-build-bundle/build-bundle-image.sh)
 export KAMEL_INSTALL_OPERATOR_IMAGE_PULL_POLICY="Always"
 
+export CAMEL_K_TEST_LOG_LEVEL="${LOG_LEVEL}"
+if [ "${LOG_LEVEL}" == "debug" ]; then
+  export CAMEL_K_TEST_MAVEN_CLI_OPTIONS="-X"
```

Comment:
I'm wondering if It could be interesting to use this env variable to set other maven CLI options. And if so, it could be `export CAMEL_K_TEST_MAVEN_CLI_OPTIONS="${CAMEL_K_TEST_MAVEN_CLI_OPTIONS} -X"`, WDYT?

Reply:
Good idea. Added.